src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1090,13 +1221,15 @@ await DownloadFileAsync(file.FilePath, destination, cancellationToken)
             ParseMode parseMode = default,
             bool disableWebPagePreview = default,
             InlineKeyboardMarkup replyMarkup = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            IEnumerable<MessageEntity> captionEntities = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.